src/main/java/com/starbucks/service/OrderService.java

```diff
@@ -17,40 +18,41 @@ public OrderService(OrderRepository orderRepository) {
         this.orderRepository = orderRepository;
     }
 
-    public String saveOrder(Order order) {
-
-        orderRepository.save ( order );
-        return order.toString ( );
+    public Order saveOrder(Order order) {
+        return orderRepository.save(order);
     }
 
-    public String updateOrder(Order order , UUID id) {
-
-        if (orderRepository.existsById ( id )) {
-            Order orderDB = orderRepository.findById ( id ).get ( );
-            orderDB.setBeverages ( order.getBeverages ( ) );
-            orderDB.setSize ( order.getSize ( ) );
-            orderDB.setSugar ( order.getSugar ( ) );
-            orderRepository.save ( orderDB );
-        }
-        return "Order " + id + " not exist";
-
+    public Order updateOrder(UUID id, Order order) {
+        Order orderDB = findById(id);
+        orderDB.setBeverages(order.getBeverages());
+        orderDB.setSize(order.getSize());
+        orderDB.setSugar(order.getSugar());
+        return orderRepository.save(orderDB);
     }
 
     public boolean existOrder(UUID id) {
-        return orderRepository.existsById ( id );
+        try {
+            return orderRepository.existsById(id);
+        } catch (RuntimeException ex) {
+            throw new ResourceAccessException("Order does not exist");
+        }
```

Comment:
you haven't checked how it's done in Todo app: https://github.com/FedosovMax/knight-todo/blob/master/src/main/java/com/knighttodo/knighttodo/service/DayService.java 
findById method